EmmadiDivyaSrujana/task2/style.css

```
@@ -0,0 +1,88 @@
+*
+
+body{
+    background-image: url(img1.jpg);
+    height: 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed;
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
```

```suggestion
   margin: 0;
}
```

Browser puts a margin around the body. This does not look good.